### ISSUE TITLE
Vocabulary from different source is not recognized by Atlas

### ISSUE THREAD

**Issue by davidhcar** [OHDSI/WebAPI#2340]
### Expected behavior
Atlas to recognize vocabulary from different source(different db/table_qualifier)

### Actual behavior
Atlas trying to find vocabulary from the CDM source(source1), rather than the Vocab source(source2). 

### Steps to reproduce behavior
Referring the https://github.com/OHDSI/WebAPI/issues/2232

create source_daimon configuration like this, 
Source 1: CDM, Achilles_Results, Achilles_Results_Temp
Source 2: Vocab

Run atlas, its looking vocab tables from CDM table_qualifier instead of vocab source(source2) table_qualifier

**Comment by davidhcar** [comment]
@chrisknoll @anthonysena

**Comment by davidhcar** [comment]
I may know the answer, it sounds only supports vocab being in a different source schema(table_qualifier) in the same database source.

**Comment by chrisknoll** [comment]
yes, we join from CDM tables to vocab tables, which cross-db joins like that is probably not handled consistently.

Sources represent the databases, and the source daimons represent schemas within the db. So, you can't set up a source to point to different databases via the CDM and Vocab schema.

**Comment by davidhcar** [comment]
Thanks @chrisknoll We have now new schema in the same database, however all the tables for Vocab in this new schema are created as Views. The atlas UI shows the vocabulary version as "access denied". I double checked the app_user access they have access to this schema. The only unknown to me is, do atlas execute as table vs name of the object in this case 'Vocabulary' ?

**Comment by chrisknoll** [comment]
Could be a permission at the database level (ie: the user/password you use to connect to that source may not have been granted select perms on the view).   To look for evidence:  go to your WebAPI log, and you should see exceptions, which may have information about permission denied on the view.

**Comment by davidhcar** [comment]
Thanks @chrisknoll, its the permission in Atlas, I explicitly need to give access to the newly added source, even though I am adding here in the UI with admin privilege.
